OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -501,8 +501,9 @@ bool HasAdequateAirUnitReloadBuildings(ActorInfo actorInfo)
 				return true;
 
 			var ammoPoolsInfo = actorInfo.TraitInfos<AmmoPoolInfo>();
+			var reloadAmmoPoolsInfo = actorInfo.TraitInfos<ReloadAmmoPoolInfo>().Where(r => r.RequiresCondition != null);
 
-			if (ammoPoolsInfo.Any(x => !x.SelfReloads))
+			if (ammoPoolsInfo.Any() && reloadAmmoPoolsInfo.Any())
```

Comment:
This is pretty concerning.
This method seems to handle how many airfields/helipads the AI builds, so changing the logic here can easily make the AI even worse ingame.
I was going to say
> You probably should check if the conditions of `ammoPoolsInfos` and `reloadAmmoPoolsInfos` intersect

but on second thought, you probably want to see if they match completely to keep the original behaviour (if you want to keep it, idk how good/bad the AI is currently regarding aircraft usage).